packages/server/tests/acceptance/rpc_batch1.spec.ts

```diff
@@ -817,6 +817,37 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
         await Assertions.assertPredefinedRpcError(error, sendRawTransaction, true, relay, [signedTx, requestId]);
       });
 
+      it('should execute "eth_sendRawTransaction" for Foundry deterministic deployment transaction', async function () {
+        // send gas money to the proxy deployer
+        const sendHbarTx = {
+          ...defaultLegacyTransactionData,
+          value: (10 * ONE_TINYBAR * 10 ** 8).toString(), // 10hbar - the gasPrice to deploy Foundry deterministic proxy contract
+          to: constants.DETERMINISTIC_DEPLOYMENT_SIGNER,
+          nonce: await relay.getAccountNonce(accounts[2].address, requestId),
+          gasPrice: await relay.gasPrice(requestId),
+        };
+        const signedSendHbarTx = await accounts[2].wallet.signTransaction(sendHbarTx);
+        await relay.sendRawTransaction(signedSendHbarTx, requestId);
+        const deployerBalance = await relay.getBalance(constants.DETERMINISTIC_DEPLOYMENT_SIGNER, 'latest', requestId);
+        expect(deployerBalance).to.not.eq(0);
+
+        // send transaction to deploy proxy transaction
+        const foundryDeterministicDeployTransactionHash = await relay.sendRawTransaction(
+          constants.DETERMINISTIC_DEPLOYER_TRANSACTION,
+          requestId,
+        );
+        const receipt = await mirrorNode.get(
+          `/contracts/results/${foundryDeterministicDeployTransactionHash}`,
+          requestId,
+        );
+        const fromAccountInfo = await global.mirrorNode.get(`/accounts/${receipt.from}`);
+
+        expect(receipt).to.exist;
+        expect(fromAccountInfo.evm_address).to.eq(constants.DETERMINISTIC_DEPLOYMENT_SIGNER);
+        // notice: the assertion below is currently blocked by the Services
+        // expect(receipt.address).to.eq(constants.DETERMINISTIC_PROXY_CONTRACT);
```

Comment:
should we open a ticket so we don't forget to uncomment this line?

Reply:
Ah I was gonna wait until the fix in the services to be included in the next releases which I believe will happen soon, then I'll uncomment this and we can merge it to main. Without the fix in the services, this feature would not work at all. I'm setting this PR as draft for early review and to test out the CIs.

Reply:
Fix is available in `v0.49.0` now. Might need a local node update